backend/src/main/java/de/bund/digitalservice/ris/norms/adapter/input/restapi/controller/ArticleController.java

```diff
@@ -293,47 +292,12 @@ public ResponseEntity<String> getArticleRender(
       @PathVariable final String language,
       @PathVariable final String subtype,
       @PathVariable final String eid,
-      @RequestParam Optional<String> atIsoDate) {
+      @RequestParam Optional<Instant> atIsoDate) {
     final String eli =
         buildEli(agent, year, naturalIdentifier, pointInTime, version, language, subtype);
 
-    if (atIsoDate.isPresent()) {
-      try {
-        DateTimeFormatter.ISO_DATE_TIME.parse(atIsoDate.get());
-      } catch (Exception e) {
-        return ResponseEntity.badRequest().build();
-      }
-
-      return loadNormUseCase
-          .loadNorm(new LoadNormUseCase.Query(eli))
-          .map(
-              norm ->
-                  applyPassiveModificationsUseCase.applyPassiveModifications(
-                      new ApplyPassiveModificationsUseCase.Query(
-                          norm, Instant.parse(atIsoDate.get()))))
-          .map(Norm::getArticles)
-          .stream()
-          .flatMap(List::stream)
-          .filter(article -> article.getEid().isPresent() && article.getEid().get().equals(eid))
-          .findFirst()
-          .map(article -> XmlMapper.toString(article.getNode()))
-          .map(
-              xml ->
-                  this.transformLegalDocMlToHtmlUseCase.transformLegalDocMlToHtml(
-                      new TransformLegalDocMlToHtmlUseCase.Query(xml, false)))
-          .map(ResponseEntity::ok)
-          .orElse(ResponseEntity.notFound().build());
-    }
-
-    return loadNormUseCase.loadNorm(new LoadNormUseCase.Query(eli)).map(Norm::getArticles).stream()
-        .flatMap(List::stream)
-        .filter(article -> article.getEid().isPresent() && article.getEid().get().equals(eid))
-        .findFirst()
-        .map(article -> XmlMapper.toString(article.getNode()))
-        .map(
-            xml ->
-                this.transformLegalDocMlToHtmlUseCase.transformLegalDocMlToHtml(
-                    new TransformLegalDocMlToHtmlUseCase.Query(xml, false)))
+    return loadArticleHtmlUseCase
+        .loadArticleHtml(new LoadArticleHtmlUseCase.Query(eli, eid, atIsoDate.orElse(null)))
```

Comment:
Should we indicate "not required" with `null`, now that we're using `null` in the use case?

Reply:
I have no idea what my comment meant. Maybe: not using `Optional<...>` in the endpoint's list of parameters.
If that's the case, I don't think it would be a better idea.